source/slang/slang-ir-specialize-function-call.cpp

```diff
@@ -620,6 +565,11 @@ struct FunctionParameterSpecializationContext
             // the arguments at the new call site, and
             // don't add anything to the specialization key.
             //
+            // if (oldIndex->getOp() == kIROp_NonUniformResourceIndex)
```

Comment:
We should not be checking in commented code.

Reply:
act